src/dcos_e2e/_common.py

```diff
@@ -76,3 +78,60 @@ def run_subprocess(
                 retcode, args, output=stdout, stderr=stderr
             )
     return CompletedProcess(args, retcode, stdout, stderr)
+
+
+def compose_ssh_command(
```

Comment:
This `_common` module is used for functions and classes which are needed by multiple modules - they are "common" or "shared" between modules. This utility is used only in the `node` module, and so it should be a private function or method there.